tests/steps/other_v2_steps.py

```diff
@@ -1446,9 +1447,12 @@ def simulate_transaction(context):
 @then("the simulation should succeed without any failure message")
 def simulate_transaction_succeed(context):
     if hasattr(context, "simulate_response"):
-        assert context.simulate_response["would-succeed"] is True
+        assert "failure-message" not in context.simulate_response["txn-groups"]
```

Comment:
Is this correct? I think something like this might be better (untested):

```python
assert all("failure-message" not in group for group in context.simulate_response["txn-groups"])
```

Reply:
see [pr response](https://github.com/algorand/py-algorand-sdk/pull/469/commits/939e1216a567006d6667ecd622965788719a41cf)